scripts/devenv-builder/configure-vm.sh

```diff
@@ -48,7 +48,19 @@ make srpm
 
 # Run MicroShift Executable > Runtime Prerequisites
 # https://github.com/openshift/microshift/blob/main/docs/devenv_rhel8.md#runtime-prerequisites
-sudo subscription-manager repos --enable rhocp-4.10-for-rhel-8-$(uname -i)-rpms --enable fast-datapath-for-rhel-8-$(uname -i)-rpms
+if curl --output /dev/null --silent --head --fail "http://download.lab.bos.redhat.com"; then
```

Comment:
Please, make it into an optional command line. The process should fail if no VPN connection is available.
I think this is necessary not to confuse non-RH users.

Reply:
Updated.